src/ApimEventProcessor/MoesifHttpMessageProcessor.cs

```diff
@@ -43,6 +43,12 @@ public MoesifHttpMessageProcessor(ILogger logger)
             _Logger = logger;
             _Logger.LogDebug("MoesifHttpMessageProcessor start..");
             var appId = ParamConfig.loadNonEmpty(MoesifAppParamNames.APP_ID);
+
+            // Set the base URI from env var, if defined.
+            var baseUri = ParamConfig.loadWithDefault(MoesifAppParamNames.BASE_URI, Moesif.Api.Configuration.BaseUri);
+            Moesif.Api.Configuration.BaseUri = baseUri;
```

Comment:
@praves77 Is the `Moesif.Api.Configuration.BaseUri` default in Moesif.Api. And it's always `moesif.net`? I am trying to understand why to reassign the variable in `Moesif.Api`

Reply:
1. Yes, it's always api.moesif.net
2. Line#49 will override the default baseUri with a different uri (to a porxy server or to moesif's dev env like `api-dev.moesif.net` )

Hope it answered your question. Happy to chat if any other questions.